rebalance/rebalance.py

```diff
@@ -212,16 +215,260 @@ def rebalance(plugin, outgoing_scid, incoming_scid, msatoshi: Millisatoshi=None,
                     excludes.append(erring_channel + '/' + str(erring_direction))
 
     except Exception as e:
-        plugin.log("Exception: " + str(e))
+        plugin.log("Exception: " + str(e), 'debug')
         return cleanup(plugin, label, payload, success_msg, e)
     return cleanup(plugin, label, payload, success_msg)
 
 
+def must_send(channel, enough_liquidity: int):
+    # liquidity is too high, must send some sats
+    min_liquidity = min(channel["msatoshi_total"] / 2, enough_liquidity)
+    their_liquidity = channel["msatoshi_total"] - channel["msatoshi_to_us"]
```

Comment:
nit: this (or similar info) is duplicated a lot in these functions, and they are called by the same caller. Maybe worth passing them all a dict with all the liquidity info ?

```python
liquidity = {
    "our": "",
    "their": "",
    "min": "",
    "ideal": {
         "our": "",
         "their": "",
    },
}
```

Reply:
Also, as cdecker points you, we should use the `..._msat` fields as they are auto intialized to Millisatoshi

Reply:
Yep, I'm planning to do that. But it will be a big change, takes some time.